src/pages/EntityDetails/App/App.js

```diff
@@ -96,6 +105,14 @@ export default function App() {
             >
               <i className="p-icon--settings"></i>Configure
             </button>
+            {showActionsPanel ? (
```

Comment:
All changes to this file are temporary to allow for easier development of the actions panel and are not visible to real users.